kalliope/core/RestAPI/FlaskAPI.py

```diff
@@ -307,6 +311,75 @@ def shutdown_server(self):
         func()
         return "Shutting down..."
 
+    @requires_auth
+    def get_mute(self):
+        """
+        Return the current trigger status
+
+        Curl test
+        curl -i --user admin:secret  -X GET  http://127.0.0.1:5000/mute
+        """
+
+        # find the order signal and call the mute method
+        for signal in SignalLauncher.get_launched_signals_list():
+            if isinstance(signal, Order):
+                data = {
+                    "mute": signal.get_mute_status()
+                }
+                return jsonify(data), 200
+
+        # if no Order instance
+        data = {
+            "error": "Mute status unknow"
+        }
+        return jsonify(error=data), 400
+
+    @requires_auth
+    def set_mute(self):
+        """
+        Set the trigger status (muted or not)
+
+        Curl test:
+        curl -i -H "Content-Type: application/json" --user admin:secret  -X POST \
+        -d '{"mute": "True"}' http://127.0.0.1:5000/mute
+        """
+
+        if not request.get_json() or 'mute' not in request.get_json():
+            abort(400)
+
+        # get mute if present
+        mute = self.get_boolean_flag_from_request(request, json_name="mute")
+
+        # find the order signal and call the mute method
+        for signal in SignalLauncher.get_launched_signals_list():
+            if isinstance(signal, Order):
+                signal.set_mute_status(mute)
+                data = {
+                    "mute": signal.get_mute_status()
+                }
+                return jsonify(data), 200
+
+        data = {
+            "error": "Cannot switch mute status"
+        }
+        return jsonify(error=data), 400
+
+    @requires_auth
+    def unmute(self):
```

Comment:
What the the difference between ```unmute``` and ```set_mute (with false in request)``` methods ?
Btw unmute is not documented

Reply:
I forgot to remove this method